bot/exts/events/hacktoberfest/hacktober-issue-finder.py

```diff
@@ -18,7 +18,7 @@
     "User-Agent": "Python Discord Hacktoberbot",
     "Accept": "application / vnd.github.v3 + json"
 }
-if GITHUB_TOKEN := Tokens.github:
+if GITHUB_TOKEN := Tokens.github.get_secret_value():
     REQUEST_HEADERS["Authorization"] = f"token {GITHUB_TOKEN}"
```

Comment:
Won't this error if `Tokens.github` is None? We should probably do something like

```suggestion
if Tokens.github:
    REQUEST_HEADERS["Authorization"] = f"token {Tokens.github.get_secret_value()}"
```

This also applies for all the other places with a similar pattern.